### PR TITLE
remove unused of

### PR DESCRIPTION
**Pull request by RomCo13**

### CODE REVIEW

**Review by sulimani55 — APPROVED** [review]

File: src/app/@ansyn/ansyn/modules/menu-items/layers-manager/services/data-layers.service.ts [hunk 1]
```diff
@@ -75,7 +75,7 @@ export class DataLayersService implements OnInit, OnDestroy {
 		}
 		return this.storageService.searchByCase<ILayer>(this.config.schema, { caseId })
 			.pipe(
-				map(layers => of(layers.slice().reverse())),
+				map(layers => layers.slice().reverse()),
```
Comment (tltk90) [inline src/app/@ansyn/ansyn/modules/menu-items/layers-manager/services/data-layers.service.ts:78]: slice is also not necessary.

**Review by tltk90 — CHANGES_REQUESTED** [review]

File: src/app/@ansyn/ansyn/modules/menu-items/layers-manager/services/data-layers.service.ts (hunk 1 above)
Comment (sulimani55) [reply src/app/@ansyn/ansyn/modules/menu-items/layers-manager/services/data-layers.service.ts:78]: i think it takes the array as a refrence, and switches its order in the store.
wouldnt it create problems on other selections?

Comment (ramtob) [reply src/app/@ansyn/ansyn/modules/menu-items/layers-manager/services/data-layers.service.ts:78]: @tltk90

Comment (sulimani55) [reply src/app/@ansyn/ansyn/modules/menu-items/layers-manager/services/data-layers.service.ts:78]: actually, i think the reverse() happends before the layers enter the store, so the slice() method might be redudent

**Pull request merged**